File: PolyPaint_ClientLourd/PolyPaint/Models/Stats/UserGameStatsHistory.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolyPaint.Models.Stats
{
    public class UserGameStatsHistory
    {
        [JsonProperty(PropertyName = "game_id")]
        public string GameID { get; set; }

        [JsonProperty(PropertyName = "timestamp")]
        public long TimeStamp { get; set; }

        [JsonProperty(PropertyName = "names")]
        public List<string> Names { get; set; }

        [JsonProperty(PropertyName = "myTeamResult")]
        public int MyTeamResult { get; set; }

        [JsonProperty(PropertyName = "otherTeamResult")]
        public int OtherTeamResult { get; set; }


        public UserGameStatsHistory()
        {
            GameID = string.Empty;
            TimeStamp = 0;
            Names = new List<string>();
            MyTeamResult = 0;
            OtherTeamResult = 0;
        }
    }
}

File: PolyPaint_Server/src/Models/UserGameStatsHistory.ts
import { Base, IBase } from "./Base";

export interface IUserGameStatsHistory extends IBase {
    game_id: string,
    timestamp: number,
    names: string[],
    myTeamResult: number,
    otherTeamResult: number,
    name: string
}

export class UserGameStatsHistory extends Base implements IUserGameStatsHistory, IBase {
    constructor(
        public game_id: string,
        public timestamp: number,
        public names: string[],
        public myTeamResult: number,
        public otherTeamResult: number,
        public name: string
    ) {
        super()
    }
}